packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1363,7 +1365,7 @@ export class ContainerRuntime
 		dataStoreAliasMap: [string, string][],
 		private readonly runtimeOptions: Readonly<Required<IContainerRuntimeOptions>>,
 		private readonly containerScope: FluidObject,
-		public readonly logger: ITelemetryLoggerExt,
+		readonly baseLogger: ITelemetryBaseLogger,
```

Comment:
Since this is public, we should restore the keyword. Our stricter linter configurations would require this.
```suggestion
		public readonly baseLogger: ITelemetryBaseLogger,
```